errbot/backends/irc.py

```diff
@@ -299,6 +313,7 @@ def __init__(self,
             self.send_public_message = rate_limited(channel_rate)(self.send_public_message)
         self._reconnect_on_kick = reconnect_on_kick
         self._pending_transfers = {}
+        self.__recently_joined_to = set()
```

Comment:
Why? this not require a external usage, is for object internal use only this variable. 
Maybe it's some specific project requirement, There is some guidelines for this? I check here: http://errbot.io/contributing.html#contributing-to-errbot-itself. Maybe i skip a additional reference?